### PR TITLE
Add guide for umami

### PR DESCRIPTION
**Pull request by thomas-kammerer**
Umami is a simple, easy to use, self-hosted web analytics solution.
https://umami.is/

### CODE REVIEW

File: source/guide_umami.rst [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+.. note::
+
+    umami is running on port 3000.
+
+.. include:: includes/web-backend.rst
+
+Configure application
+---------------------
+
+For more information check the umami_ documentation.
```
Comment (luto) [inline source/guide_umami.rst:120]: Please add the default username/password as well as a warning box to change it immediately somewhere in the guide. I was kind of lost at the unexpected login screen :sweat_smile:

**Review by luto — CHANGES_REQUESTED** [review]
nicely done! I added a few comments, most of which you can fix by accepting the suggested change. Let me know what you think! :)

**Comment by thomas-kammerer** [comment]
Thanks, I made the changes accordingly to your comments :)

File: source/guide_umami.rst [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+ [isabell@stardust umami]$ mysql -u isabell -p isabell_umami < sql/schema.mysql.sql 
+ [isabell@stardust umami]$
+
+Enter your MySQL password.
+
+Use your favorite editor to create ~/umami/.env with the following content:
+
+.. code-block:: ini
+
+DATABASE_URL=mysql://isabell:mypassword@localhost:3306/umami
```
[the remark this answers is not in the document]
Comment (luto) [reply source/guide_umami.rst]: the new `isabell_umami:mypassword@localhost:3306/umami` changed the username to `isabell_umami`, not the database name.

**Comment by luto** [comment]
thank you! :)

**Pull request merged**